### PR TITLE
[AWS] [ACM] add generators

### PR DESCRIPTION
**Pull request by nagypeterjob**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

- [x] add-tags-to-certificate
- [x] delete-certificate
- [x] describe-certificate
- [x] export-certificate
- [x] get-account-configuration
- [x] get-certificate
- [x] import-certificate
- [x] list-certificates
- [x] list-tags-for-certificate
- [x] put-account-configuration
- [x] remove-tags-from-certificate
- [x] renew-certificate
- [x] request-certificate
- [x] resend-validation-email
- [x] update-certificate-options
- [x] wait

### CODE REVIEW

**Comment by nagypeterjob** [comment]
The same old `listFIles` & `listBlobs` generators which ya'll know too well

File: dev/aws/acm.ts [hunk 1]
```diff
@@ -522,6 +745,8 @@ export const completionSpec: Fig.Spec = {
             "The domain name that you want ACM to use to send you emails so that you can validate domain ownership.",
           args: {
             name: "list",
+            variadic: true,
+            description: "DomainName=string,ValidationDomain=string",
```
Comment (QuiiBz) [inline dev/aws/acm.ts:749]: Is this argument really variadic? The description doesn't look like it is.

**Review by QuiiBz — COMMENTED** [review]
Only small feedback to resolve, the rest seems good to me. Soon, you won't need to duplicate code across spec - it's going to be a lot cleaner!

File: dev/aws/acm.ts (hunk 1 above)
Comment (nagypeterjob) [reply dev/aws/acm.ts:749]: Lists are usually variadic in the aws cli universe. Check out the example:

Shorthand Syntax:

`DomainName=string,ValidationDomain=string ...`

JSON Syntax:
```json
[
  {
    "DomainName": "string",
    "ValidationDomain": "string"
  }
  ...
]
```

The "..." means it is variadic

Comment (QuiiBz) [reply dev/aws/acm.ts:749]: Thanks for the clarification!

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Pull request merged**